dask/async.py

```diff
@@ -429,6 +417,10 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
     """
     assert queue
 
+    if callbacks is None:
+        callbacks = _globals['callbacks']
+    callbacks = setup_callbacks(callbacks)
```

Comment:
That certainly is simpler. I assumed that most `callbacks` objects would have only one or two functions, with the remainder as None. I did it this way to avoid iterating through many empty callbacks at each tick. However, I doubt there will ever be more than a handful of these, so that may have been unnecessary. idk.